source/index.html.md

```diff
@@ -808,6 +808,42 @@ object][ApplePay-PaymentToken] for more information.
   subsequent-in-series authorization.
 </p>
 
+Additionally, an Apple Pay authorization can be created using raw values from
+a payment token:
```

Comment:
Does it belong here to mention that the TP should reach out to ensure that they're allowed to use `applepay[raw]`?

What about listing the requirements the the TP should enforce? (Check for signature, for age, and other stuff mentioned in https://developer.apple.com/library/archive/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html)

Reply:
https://github.com/clearhaus/gateway-api-docs/pull/137/commits/be79759f58f5875fa869c569489da563c6e1304f

I moved the alert box down because its notices are also relevant for raw Apple Pay. Both non-raw and raw require that a TP is allowed to use AP.